templates/bundles/EasyAdminBundle/layout.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends '@!EasyAdmin/layout.html.twig' %}
```

Comment:
User crudController not display, it seem `EasyAdminBundle/layout.html.twig` need to by replace by `EasyAdminBundle/welcome.html.twig`.

To update the design of EasyAdmin copy and modify the elements of this path :
`vendor\easycorp\easyadmin-bundle\src\Resources\views`